### DISCUSSION TITLE
A couple more user.js options

### DISCUSSION THREAD

**Discussion by fabcard** [farag2/Mozilla-Firefox#3] [Ideas]
Hello there, @farag2 
Could you add into user.js the options to enable/disable "Ask to save logins and passwords for websites" and an option to set the "Default Search Engine" to DuckDuckGo or any other engine, please?
Thanks in advance.

**Comment by farag2** [comment]
Only here for you. 😸 

```javascript
user_pref("signon.rememberSignons", false);
user_pref("browser.urlbar.placeholderName", "DuckDuckGo");
```

**Reply by fabcard** [reply]
Thank you, friend :)
Have a nice week.

**Comment by fabcard** [comment]
@farag2 
Sorry for bothering you, friend.
I added those lines at the end of user.js file. The option that asks to save logins and password is working fine, but the other to set DuckDuckGo (or any other) as default is not working.
Could you help me when you have a little time to spare, please?
Again, thank you.

**Comment by farag2** [comment]
I forgot that Firefix stores this setting in the `%APPDATA\Mozilla\Firefox\Profiles\<profilename>\search.json.mozlz4` file. You need to configure all the search engines, set one as a default and back up this file. There is no way to configure it via user.js

**Reply by fabcard** [reply]
Ok friend. Thank you.
